test/Microsoft.OpenApi.Tests/Validations/OpenApiInfoValidationTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OpenApi.Models;
+using Xunit;
+
+namespace Microsoft.OpenApi.Validations.Tests
+{
+    public class OpenApiInfoValidationTests
+    {
+        [Fact]
+        public void ValidateFieldIsRequiredInInfo()
+        {
+            // Arrange
+            IEnumerable<ValidationError> errors;
+            OpenApiInfo info = new OpenApiInfo();
+
+            // Act
+            bool result = info.Validate(out errors);
+
+            // Assert
+            Assert.False(result);
+            Assert.NotNull(errors);
+
+            Assert.Equal(new[] { "The field 'url' in 'info' object is REQUIRED.",
+                "The field 'version' in 'info' object is REQUIRED."}, errors.Select(e => e.ErrorMessage));
```

Comment:
These strings in all test classes should come from the main project, so that we don't need to keep track of the strings in two places.

Reply:
Same as others:

Yes. I can use the

Assert.Equal(String.Format(SRResource.xxxxx, typeof(xxx).FullName), exception.Message);

However, it looks not good than the plain text in the test project. Because it can help us verify the error message thrown as expected.  What do you think?

---
In reply to: [157650687](https://github.com/Microsoft/OpenAPI.NET/pull/152#discussion_r157650687) [](ancestors = 157650687)